contracts/PetPark.sol

```diff
@@ -1,7 +1,92 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
-
+import "hardhat/console.sol";
 
 contract PetPark {
+    address public owner;
+
+    mapping(string => uint) public AnimalType;
+
+    mapping(uint => uint) public animalCounts;
+
+    mapping(address => User) public Users;
+
+	mapping(string => uint8) public Gender;
```

Comment:
Weird, I already have that plugin installed globally. 
<img width="467" alt="image" src="https://user-images.githubusercontent.com/11972548/169108519-bd5717a8-6e60-47b3-941a-df37808b37ed.png">

Maybe it's because I also had the Solidity extension by Juan Blanco enabled?

It also looks fine in the editor so not sure what the issue could be?

Reply:
@shubham-kanodia Fixed this by disabling Juan Blanco extension!